integration/fabric/iou/commands.go

```diff
@@ -75,15 +75,15 @@ func CheckLocalMetrics(ii *integration.Infrastructure, user string, viewName str
 	gomega.Expect(metrics).NotTo(gomega.BeEmpty())
 
 	var sum float64
-	for _, m := range metrics["fsc_view_operations"].GetMetric() {
+	for _, m := range metrics["fsc_view_services_view_calls_operations"].GetMetric() {
```

Comment:
Please, introduce a constant for the value `"fsc_view_services_view_calls_operations"`